packages/appeals-service-api/src/routes/documentMetadata.js

```diff
@@ -6,10 +6,11 @@ router.get('/:caseref', async (req, res) => {
 	let statusCode = 200;
 	const caseRef = req.params.caseref;
 	const documentType = req.query.documenttype;
+	const returnMultipleDocuments = req.query?.returnMultipleDocuments ?? false;
```

Comment:
Maybe do something here to cast this as boolean so we can operate on it as a boolean down the chain